grpc-examples/golang/README.md

```diff
@@ -0,0 +1 @@
+# gRPC Rust Client
```

Comment:
how feasible it is to move the rust examples here? I think it is not feasible. Shouldn't it live in a dedicated repo (similar to go, node)?